modules/ROOT/pages/maven-reference.adoc

```diff
@@ -45,8 +45,39 @@ In this case, it is a _user_ `settings.xml` file that is relevant only for a spe
 
 On a system with both global and user `settings.xml` files, the settings from both files become merged. In case there are duplicate settings, the user's settings take precedence over the global settings.
 
+== Mule Repositories
+
+When you create a new Mule project from Anypoint Studio, the minimum required Mule repositories are automatically added to the `pom.xml` file. If you are creating the `pom.xml` file manually, add the following repositories:
```

Comment:
Should "from Anypoint Studio" be "in Anypoint Studio"?
What is the difference between "minimum required Mule repositories" (which sounds a bit awkward) and "required Mule repositories"? If there is no difference, try using the latter.